### ISSUE TITLE
FFmpeg

### ISSUE THREAD

**Issue by AnjanaNiranjan** [RyanZotti/Self-Driving-Car#119]
How do I install FFmpeg?

**Comment by javierrcc522** [comment]
https://www.hackster.io/whitebank/rasbperry-pi-ffmpeg-install-and-stream-to-web-389c34
this might help but be aware that ffserver has been removed from newer release so you are going to figure that one out

**Comment by RyanZotti** [comment]
Yes, ffmpeg is definitely a huge pain to install. 2 years ago when I was trying to install it I tried countless different approaches (including skipping ffmpeg altogether). I tried so many things that I didn’t know which approach worked because my system had become a huge mess. Once ffmpeg worked though I never had to touch it again, so I’ve never had to relive the install pain and therefore haven’t had an opportunity to re-document it. 

It’s likely that you could just use another open source video streaming library (there are probably newer and better ones available). You’ll probably have to change a small portion of the python code in my repo, but it might save you the ffmpeg install headache. 

Sorry I couldn’t be of more help. I honestly hate ffmpeg. Let me know if you come across something better that you would recommend to others.

**Comment by Nijisakai** [comment]
`wget https://ffmpeg.org/releases/ffmpeg-3.4.4.tar.bz2`
inside of 
`git clone git://source.ffmpeg.org/ffmpeg.git`
will do the trick.
